src/lib/orionld/rest/orionldServiceLookup.cpp

```diff
@@ -45,7 +45,7 @@
 //
 //  33 - 9 == 24
 //
-#define MAX_CHARS_BEFORE_WILDCARD 24
+#define MAX_CHARS_BEFORE_WILDCARD 26
```

Comment:
The comment says 24 but the definition says 26 ... 
That's a bit confusing.
Please edit the comment accordingly